components/style/matching.rs

```diff
@@ -437,55 +439,50 @@ trait PrivateMatchMethods: TElement {
         _restyle_hint: RestyleHint,
         _important_rules_changed: bool,
     ) {
-        use crate::animation;
-
         let this_opaque = self.as_node().opaque();
-        let mut expired_transitions = vec![];
         let shared_context = context.shared;
+        let mut animation_states = shared_context.animation_states.write();
+        let mut animation_state = animation_states
+            .remove(&this_opaque)
+            .unwrap_or_else(Default::default);
```

Comment:
`unwrap_or_default()`?

Reply:
Thanks! I didn't know this existed. It makes more sense here.